web/massastation/src/pages/Store/StoreSection/StorePlugin.tsx

```diff
@@ -36,13 +41,24 @@ function StorePlugin(props: StorePluginProps) {
     }
   }, [isInstallSuccess]);
 
+  const argsStoreError = [
```

Comment:
These args are not related to Error/Warning only. We have all topAction here, even download one. It should call topActions only.

Reply:
agreed